src/checkers/lines.rs

```diff
@@ -33,6 +35,12 @@ pub fn check_lines(contents: &str, settings: &Settings, autofix: flags::Autofix)
                 checks.push(check);
             }
         }
+
+        if enforce_blanket_type_ignore {
+            if let Some(check) = blanket_type_ignore(lineno, line) {
```

Comment:
I think we should _either_ only run this on lines with comments (`directives.commented_lines`), or move this into `checkers/tokens.rs` and run it directly on `Tok::Comment` nodes (the downside of which is that we have to use `Locator` to get the comment).